specifications/api/swagger.yaml

```diff
@@ -71,6 +73,12 @@ x-blockId: &blockId
   minLength: 64
   example: 3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3fe6c52041a
 
+x-absoluteSlot: &absoluteSlot
+  type: integer
+  description: The number of slots since genesis of the blockchain.
```

Comment:
Maybe it's just me, but "the number of slots since genesis" confuses me slightly because I wonder if the moment right after genesis is slot 0 or 1.

But it should be 0:

```
|  ------- slot 0 -----------| ---------- slot 1 --------- | ... 
^ GENESIS
```

```
--------
slot 0...
started at: 2017-09-23 21:44:51 UTC
ended at: 2017-09-23 21:45:11 UTC
--------
Genesis date: 21:44:51 UTC
```

Reply:
I wonder if some concrete examples could be nice in the descriptions, e.g.
- `slot_number`: Ranges from 0 to 43199 in each Shelley-era epoch.
- `absolute_slot_number`: ranges from Y to Z in epoch 218. The slot with `absolute_slot_number=0` began on 2017-09-23 21:44:51 UTC and lasted for 20 seconds.

or perhaps that is redundant / too much noise.

Reply:
OK I reworded to say it is zero-based.